app.py

```diff
@@ -47,8 +47,8 @@ def inprogress_item(id):
     @app.route('/tasks/<id>', methods=['GET'])
     def get_item(id):
         singleitem = trello_items.get_single_item(id)
-        print(singleitem)
-        return render_template("index.html", singleitem=singleitem)
+        #print(singleitem)
```

Comment:
Don't keep commented out code.

But actually this whole function/route is unused and should be deleted. The render_template line is also wrong because it's setting ViewModel rather than view_model.